pkg/framework/test/integration/integration_test.go

```diff
@@ -17,7 +17,7 @@ var _ = Describe("The Testing Framework", func() {
 		var err error
 		var fixtures *test.Fixtures
 
-		fixtures, err = test.NewFixtures(defaultPathToEtcd)
+		fixtures, err = test.NewFixtures()
```

Comment:
Isn't this hiding too much now? I can't configure the binaries, which means I can't test against multiple versions?  OK I can change the binaries in the env location, or change the env. 
Should we have a config that let's one change the default?

Reply:
You can configure the binaries by:
- Putting different binaries in the default path `assets/bin` (relative to the test framework package)
- Changing environment variables in your test environment (eg `TEST_ASSET_ETCD`)
- Changing the pathfinder in code:
```
fixtures = test.NewFixtures()
apiServer = test.NewAPIServer() // currently this takes more params, but we're working on it
apiServer.PathFinder = func(_ string) string {
  return "/my/path/to/apiserver"
}
fixtures.APIServer = apiServer
fixtures.Start()
```